app/messaging/receivers.js

```diff
@@ -20,12 +21,13 @@ process.on('SIGINT', async () => {
 })
 
 module.exports = {
-  startUpdateEligibility: async function (eligibilityAction) {
-    updateEligibilityReceiver = new MessageReceiver(msgCfg.updateEligibilityQueue, eligibilityAction)
-    await updateEligibilityReceiver.connect()
+  startUpdateEligibility: async function () {
+    updateEligibilityReceiver = new MessageReceiver(msgCfg.updateEligibilityQueue, (msg) => {})
```

Comment:
I don't think `msg` is needed, it isn't used in the empty function. I don't know wether it would be worth adding a log to capture it has been executed?